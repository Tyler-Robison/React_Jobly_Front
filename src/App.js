import React from "react";
import './Styles/App.css';
import Utility from './Utility';

function App() {
  return (
    <div className="App">
    <Utility />
    </div>
  );
}

export default App;
