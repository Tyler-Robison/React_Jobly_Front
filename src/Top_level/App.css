.App {
  text-align: center;
}

body {
  background-color: rgb(190, 243, 250);
}
