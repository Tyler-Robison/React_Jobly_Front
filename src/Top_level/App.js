import React from "react";
import './App.css';
import Utility from './Utility';

function App() {
  return (
    <div className="App">
    <Utility />
    </div>
  );
}

export default App;
