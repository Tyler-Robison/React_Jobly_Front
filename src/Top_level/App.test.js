import { render, screen } from '@testing-library/react';
import App from './App';
import { MemoryRouter } from "react-router-dom";


it("renders without crashing", function () {
  <MemoryRouter>
    <App />
  </MemoryRouter>
});

it("matches snapshot", function () {
  const container = render(
      <MemoryRouter initialEntries={['/']}>
          <App />
      </MemoryRouter>
  );
  expect(container.asFragment()).toMatchSnapshot();
});
