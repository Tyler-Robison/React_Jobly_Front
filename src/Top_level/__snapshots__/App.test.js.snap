// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <div>
        <ul>
          <li>
            <a
              aria-current="page"
              class="nav-link active"
              href="/jobly"
            >
              Jobly
            </a>
          </li>
          <li>
            <a
              class="nav-link"
              href="/signup"
            >
              Signup
            </a>
          </li>
          <li>
            <a
              class="nav-link"
              href="/login"
            >
              Login
            </a>
          </li>
        </ul>
      </div>
      <h3 />
      <h1>
        Jobly
      </h1>
      <h3>
        All the jobs in one convenient place
      </h3>
      <p />
    </div>
  </div>
</DocumentFragment>
`;
